cli/src/subcommand.rs

```diff
@@ -147,13 +154,13 @@ pub fn check_and_comment_on_pr(
                 get_github_private_key(github_private_key, github_private_key_base64)?;
             let gh = app::GitHub::new(&gh_private_key, github_app_id, github_install_id)?;
 
-            return Ok(comment_on_pr(
+            comment_on_pr(
                 &gh,
                 &github_repo_owner,
                 &github_repo_name,
                 github_pr_number,
                 &comment_body,
-            )?);
+            )?;
         }
     }
     if let Some(github_token) = github_token {
```

Comment:
```suggestion
    else if let Some(github_token) = github_token {
```

so we don't have a case where we could have commont_on_pr run twice

Reply:
I missed that, thanks for mentioning. Done in fcc2dca7a72ca33aae5592b8d1daf3322701906e.